Day6/index.html

```diff
@@ -0,0 +1,88 @@
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+
+    <div class="container">
+
+      <!-- Welcome Text -->
+      <div class="row justify-content-center">
+        <p id="welcome-text">" Hang man Game "</p>
+      </div>
+
+      <!-- Image field-->
+      <div class="row justify-content-center">
+        <img src="" alt="image" id="status-image">
+      </div>
+
+      <!-- Turns field-->
+      <div class="row justify-content-center">
+        <span class="turn-display">Number of Turns : <span id="num-turn">7</span></span>
+      </div>
+
+      <br>
+
+      <!-- Word field -->
+      <div class="row justify-content-center">
+        <span id="player-guess-display">TEXT HERE</span>
+      </div>
+
+      <br>
+
+      <!-- Button field -->
+      <div class="row justify-content-center" id="row-1">
+        <button type="button" class="btn btn-outline-primary" id="a">A</button>
+        <button type="button" class="btn btn-outline-secondary" id="b">B</button>
+        <button type="button" class="btn btn-outline-success" id="c">C</button>
+        <button type="button" class="btn btn-outline-danger" id="d">D</button>
+        <button type="button" class="btn btn-outline-warning" id="e">E</button>
+        <button type="button" class="btn btn-outline-info" id="f">F</button>
+        <button type="button" class="btn btn-outline-dark" id="g">G</button>
+      </div>
+
+      <div class="row justify-content-center" id="row-2">
+        <button type="button" class="btn btn-outline-primary" id="h">H</button>
+        <button type="button" class="btn btn-outline-secondary" id="i">I</button>
+        <button type="button" class="btn btn-outline-success" id="j">J</button>
+        <button type="button" class="btn btn-outline-danger" id="k">K</button>
+        <button type="button" class="btn btn-outline-warning" id="l">L</button>
+        <button type="button" class="btn btn-outline-info" id="m">M</button>
+        <button type="button" class="btn btn-outline-dark" id="n">N</button>
+      </div>
+
+      <div class="row justify-content-center" id="row-3">
+        <button type="button" class="btn btn-outline-primary" id="o">O</button>
+        <button type="button" class="btn btn-outline-secondary" id="p">P</button>
+        <button type="button" class="btn btn-outline-success" id="q">Q</button>
+        <button type="button" class="btn btn-outline-danger" id="r">R</button>
+        <button type="button" class="btn btn-outline-warning" id="s">S</button>
+        <button type="button" class="btn btn-outline-info" id="t">T</button>
+        <button type="button" class="btn btn-outline-dark" id="u">U</button>
```

Comment:
Do not hard code all elements. Try to find a pattern, and add it using loops with jQuery. Only one loop is enough to create all these 40 lines of code.